src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
Think about which variables should be local in the method and which should be class-level. `new ColorSupplier()` object will be created on each `getRandomBall()` call. How can you avoid this?